cordova-sdk/scripts/inject-config.js

```diff
@@ -134,32 +136,38 @@ function prepareAndroid(context, config) {
 }
 
 function createJsonWithDefaultValues(
-  optimoveCredentials, 
-  optimoveMobileCredentials , 
-  inAppConsentStrategy, 
-  enableDeferredDeepLinking,
-  android) {
+    optimoveCredentials,
+    optimoveMobileCredentials,
+    inAppConsentStrategy,
+    enableDeferredDeepLinking,
+    delayedInitialization,
+    featureSet,
+    android
+) {
   return {
     OPTIMOVE_CREDENTIALS:
-      !isEmpty(optimoveCredentials) &&
-      isString(optimoveCredentials)
-        ? optimoveCredentials
-        : "",
+        !isEmpty(optimoveCredentials) && isString(optimoveCredentials)
+            ? optimoveCredentials
+            : "",
     OPTIMOVE_MOBILE_CREDENTIALS:
-      !isEmpty(optimoveMobileCredentials) &&
-      isString(optimoveMobileCredentials)
-        ? optimoveMobileCredentials
-        : "",
+        !isEmpty(optimoveMobileCredentials) && isString(optimoveMobileCredentials)
+            ? optimoveMobileCredentials
+            : "",
     IN_APP_STRATEGY: inAppConsentStrategy,
-    ENABLE_DEFERRED_DEEP_LINKING:
-      enableDeferredDeepLinking === true,
-    ANDROID_PUSH_NOTIFICATION_ICON_NAME: 
-      android !== undefined &&
-      !isEmpty(android.pushNotificationIconName) &&
-      isString(android.pushNotificationIconName)
-        ? android.pushNotificationIconName
-        : ""
- }
+    ENABLE_DEFERRED_DEEP_LINKING: enableDeferredDeepLinking === true,
+    ENABLE_DELAYED_INITIALIZATION: delayedInitialization && delayedInitialization.enable === true,
+    DELAYED_INITIALIZATION_REGION: delayedInitialization && isString(delayedInitialization.region)
+        ? delayedInitialization.region
+        : "",
+    ANDROID_PUSH_NOTIFICATION_ICON_NAME:
+        android !== undefined &&
+        !isEmpty(android.pushNotificationIconName) &&
+        isString(android.pushNotificationIconName)
+            ? android.pushNotificationIconName
+            : "",
+    DELAYED_INITIALIZATION_ENABLE_OPTIMOVE: featureSet && featureSet.enableOptimove === true,
```

Comment:
simpler to have in optimove config `featureSet: [optimove, optimobile]`? later can add pref center with changing less